### ISSUE TITLE
GU0030 should not warn here

### ISSUE THREAD

**Issue by JohanLarsson** [JohanLarsson/Gu.Analyzers#79]
```c#
public class Foo
{
    private Disposable[] disposables;

    public Foo()
    {
        var items = new Disposable[2];
        for (var i = 0; i < 2; i++)
        {
            var item = new Disposable();
            items[i] = item;
        }

        this.disposables = items;
    }
}
```

**Comment by JohanLarsson** [comment]
Fixed, forgot to close with commit.
